nwbinspector/checks/time_series.py

```diff
@@ -0,0 +1,85 @@
+"""Authors: Cody Baker, Ben Dichter, and Ryan Ly."""
+import numpy as np
+
+import pynwb
+
+# from ..tools import all_of_type
+from ..register_checks import register_check, Importance, Severity, InspectorMessage
+from ..utils import check_regular_series
+
+
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=pynwb.TimeSeries)
+def check_regular_timestamps(time_series: pynwb.TimeSeries, time_tol_decimals=9, gb_severity_threshold=1.0):
+    """If the TimeSeries uses timestamps, check if they are regular (i.e., they have a constant rate)."""
+    if time_series.timestamps is not None and check_regular_series(
+        series=time_series.timestamps, tolerance_decimals=time_tol_decimals
```

Comment:
Just for consistency, I would suggest renaming ``time_tol_decimals`` to ``time_tolerance_decimals``